src/com/owncloud/android/operations/RefreshFolderOperation.java

```diff
@@ -349,11 +349,6 @@ private void synchronizeData(ArrayList<Object> folderAndFiles, OwnCloudClient cl
         // get 'fresh data' from the database
         mLocalFolder = mStorageManager.getFileByPath(mLocalFolder.getRemotePath());
 
-        // parse data from remote folder 
-        OCFile remoteFolder = FileStorageUtils.fillOCFile((RemoteFile) folderAndFiles.get(0));
```

Comment:
The ```OCFile``` instance you are retrieving in line 440 is the same you have in the field ```mLocalFolder```; that's why its local properties are copied into ```remoteFolder``` right in the code where you removed it. The idea is not doing an extra call to the database, since we consider at this moment that ```mLocalFolder``` is "fresh enough".

If you miss a local property, such as availableOfflineStatus (favorite before), it could just be copied in the same way.